tasks/copy_to_archive/README.md

```diff
@@ -223,31 +224,40 @@ The output of this lambda is a dictionary with a `granules` and `copied_to_orca`
 
 ## Configuration
 
-As part of the [Cumulus Message Adapter configuration](https://nasa.github.io/cumulus/docs/workflows/input_output#cma-configuration) 
-for `copy_to_archive`, the `excludedFileExtensions`, `s3MultipartChunksizeMb`, `providerId`, `executionId`, `collectionShortname`, `collectionVersion` and `defaultBucketOverride` keys must be present under the 
-`task_config` object as seen below. Per the [config schema](https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_archive/schemas/config.json), 
-the values of the keys are used the following ways. The `provider` key should contain an `id` key that returns the provider id from Cumulus. The `cumulus_meta` key should contain an `execution_name` key that returns the step function execution ID from AWS. 
+The `providerId`, `executionId`, `collectionShortname`, and `collectionVersion` and keys must be present under the 
```

Comment:
change to 
The `providerId`, `executionId`, `collectionShortname`, and `collectionVersion` keys

Reply:
Whoops. Fixed.